hack/aks/Makefile

```diff
@@ -373,6 +356,9 @@ linux-cniv1-up: rg-up overlay-net-up ## Bring up a Linux CNIv1 cluster
 		--no-ssh-key \
 		--yes
 	@$(MAKE) set-kubeconf
+ifeq ($(OS),windows)
```

Comment:
Maybe a cluster having Windows nodes is...something we support and expect to happen and all of our test clusters should include?
Instead of being mostly ignored except for during the tests which specifically excercise Windows things?
WDYT @behzad-mir